### ISSUE TITLE
Loss_bbox and Loss_dfl are both 0

### ISSUE THREAD

**Issue by sosgu** [AILab-CVC/YOLO-World#582]
I am executing on the terminal
CUDA_VISIBLE_DEVICES=1 python tools/train.py configs/finetune_coco/yolo_world_v2_l_vlpan_bn_sgd_1e-3_40e_8gpus_finetune_coco.py --work-dir log --amp --resume

After command, display after training
`style="text-decoration-style:single">INFO</u></font> - Epoch(train) [25][50/90]  base_lr: 1.0000e-03 lr: 4.3075e-04  eta: 0:09:11  time: 0.3959  data_time: 0.0126  memory: 5473  grad_norm: 0.0283  loss: 0.0016  loss_cls: 0.0016  loss_bbox: 0.0000  loss_dfl: 0.0000`

I searched for relevant web pages online and said that I need to keep the tags in the JSON file consistent, but after modifying it, the effect is still the same. Therefore, I would like to consult with everyone here on how to solve this problem;

The categories in my training set are
    "categories": [
        {
            "id": 0,
            "name": "head",
            "supercategory": "head"
        }
    ]

The categories in the test set are
    "categories": [
        {
            "id": 0,
            "name": "head",
            "supercategory": "head"
        }
    ]
The content in coco_class_texts.json is
[["head"]]

Finally, in the yolo_corld_v2_1-vlpan-bn_sgd_1e-3_40e_8gpus_inetune_como.py file, the following is included:
metainfo = dict(classes=("head"))

coco_train_dataset = dict(_delete_=True,
                          type='MultiModalDataset',
                          dataset=dict(
                              type='YOLOv5CocoDataset',
                              metainfo=metainfo,

Only a portion of the code has been intercepted. Could you please provide guidance, experts

**Comment by 2950375033** [comment]
你好，我遇到了同样的问题，请问你解决了吗？
